data/korath/korath ships.txt

```diff
@@ -851,6 +851,213 @@ ship "Korath World-Ship" "Korath World-Ship C"
+		angle 180
+	"steering engine" -152.5 -270
+		angle 45
+		right
+		zoom 0.6
+	"steering engine" 152.5 -270
+		angle 315
+		left
+		zoom 0.6
+	"steering engine" -172 169
+		angle 135
+		left
+		zoom 0.6
+	"steering engine" 172 169
+		angle 225
+		right
+		zoom 0.6
+	"steering engine" -135.5 444
+		zoom 1.2
+		right
+	"steering engine" 135.5 444
+		zoom 1.2
+		left
+	engine 0 481.5
+		zoom 1.7
+	engine -135.5 444
+		zoom 1.6
+	engine 135.5 444
+		zoom 1.6
+	engine -88.5 430.5
+		zoom 0.7
+	engine 88.5 430.5
+		zoom 0.7
+	engine -77 436
+		zoom 0.7
+	engine 77 436
+		zoom 0.7
+	engine -46 454.5
+		zoom 0.8
+	engine 46 454.5
+		zoom 0.8
+	turret -19.5 -438.5 "Korath Banisher"
+	turret 19.5 -438.5 "Korath Warder"
+	turret -178 -332 "Korath Warder"
+	turret 178 -332 "Korath Banisher"
+	turret -186 -285.5 "Korath Banisher"
+	turret 186 -285.5 "Korath Warder"
+	turret -131.5 -173.5 "Korath Heaver"
+	turret 131.5 -173.5 "Korath Inferno"
+	turret -131.5 -156 "Korath Inferno"
+	turret 131.5 -156 "Korath Warder"
+	turret 46 -12 "Korath Warder"
+	turret -16.5 16 "Korath Banisher"
+	turret 62 128 "Korath Heaver"
+	turret -75.5 162.5 "Firestorm Turret"
+	turret -204.5 200 "Korath Banisher"
+	turret 204.5 200 "Korath Warder"
+	turret -192.5 257.5 "Korath Warder"
+	turret 192.5 257.5 "Korath Banisher"
+	turret -72.5 338.5 "Korath Heaver"
+	turret 72.5 338.5 "Korath Inferno"
+	turret -95.5 365.5 "Korath Inferno"
+	turret 95.5 365.5 "Korath Banisher"
+	gun -170 -104 "Firelight Missile Bank"
+		angle 315
+	gun 170 -104 "Firelight Missile Bank"
+		angle 45
+	gun -178 -84 "Firelight Missile Bank"
+		angle 290
+	gun 178 -84 "Firelight Missile Bank"
+		angle 70
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	bay "Fighter" -167.5 -420
+		angle 315
+	bay "Fighter" 167.5 -420
+		angle 45
+	bay "Fighter" -179.5 80
+		angle 45
+		back
+	bay "Fighter" 179.5 80
+		angle 315
+		back
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Korath Havens are massive nomadic habitats that hold all that is left of their people."
```

Comment:
I realise fixing descs is like my job but I'm so tired I don't want to lose track

```suggestion
	description "Korath Havens are massive nomadic habitats that hold much of their remaining population."
```